### PR TITLE
Swagger - Doc 0.1.0

### PR DESCRIPTION
**Pull request by Tower450**
Simple YAML file for swagger API documentation.

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+      }
+
+
+  LikemarkFirstChildrenResponse:
```
Comment (vlnk) [inline swagger.yml]: it is the same as the LikemarkListResponse, you should factorize it

File: swagger.yml [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+  /likemark/getWithFirstChildren/{id}:
+    get:
+        summary: Get a likemark with his children by his id.
+        parameters:
+        - in: path
+          name: id
+          type: string
+          required: true
+          description: Unique id of the likemark.
```
Comment (vlnk) [inline swagger.yml:69]: the children object is not documented

Comment (vlnk) [reply swagger.yml:69]: maybe there is an object called LikemarkWithFirstChildrenResponse

**Review by vlnk — COMMENTED** [review]
some models are incoherents

File: swagger.yml (hunk 1 above)
Comment (Tower450) [reply swagger.yml]: will be fixed.

File: swagger.yml (hunk 2 above)
Comment (Tower450) [reply swagger.yml:69]: children object is not a parameter but an attribute of the response.

Comment (Tower450) [reply swagger.yml]: - [ ] will be fixed

**Pull request merged**